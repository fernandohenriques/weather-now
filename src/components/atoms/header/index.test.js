import React from 'react';
import { shallow } from 'enzyme';
import Header from './index';

describe('<Header />', () => {
  it('Snapshot testing', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });

  it('Should find semantic tags Header and Figure', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.exists()).toBeTruthy();
    expect(wrapper.find('header')).toHaveLength(1);
    expect(wrapper.find('figure')).toHaveLength(1);
  });
});
