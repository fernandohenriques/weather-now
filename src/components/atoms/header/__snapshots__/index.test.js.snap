// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Snapshot testing 1`] = `
<header
  className="header"
>
  <figure>
    <img
      alt="App logo"
      src="logo.svg"
      title="App logo"
    />
  </figure>
</header>
`;
